app/services/service_provider_seeder.rb

```diff
@@ -50,4 +51,20 @@ def write_service_provider?(config)
 
     !is_production_or_has_a_restriction || (restrict_env == deploy_env)
   end
+
+  def check_for_missing_sps
+    return unless %w[prod staging].include? deploy_env
+
+    sps_in_db = ServiceProvider.pluck(:issuer)
+    sps_in_yaml = service_providers.keys
+    extra_sps = sps_in_db - sps_in_yaml
+    return unless extra_sps.count.positive?
```

Comment:
I think we can rewrite this to be a positive check instead of a negative check?

```suggestion
    return if extra_sps.empty?
```

Reply:
Sure!